com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -504,6 +514,252 @@ internal void InvokeBehaviourOnGainedOwnership()
             }
         }
 
+        internal void InvokeBehaviourOnNetworkObjectParentChanged(NetworkObject parentNetworkObject)
+        {
+            for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
+            {
+                ChildNetworkBehaviours[i].OnNetworkObjectParentChanged(parentNetworkObject);
+            }
+        }
+
+        private bool m_IsReparented;
+        private ulong? m_LatestParent;
```

Comment:
Worth calling out what this variable means.  Is this simply the parent that I think I have now?

Reply:
good shout, will comment inline.

Reply:
done.